Packs/CyrenInboxSecurity/Playbooks/Cyren_Inbox_Security_playbook.yml

```diff
@@ -0,0 +1,752 @@
+description: 'Processes Cyren Incidents, sets resolutions, and applies remediations
+  to end-user mailboxes. '
+id: be5080ec-ac04-459c-8eb0-0f0323c7af50
```

Comment:
The ID and the name should we the same, please replace the ID with the value as the name